flink-runtime/src/main/java/org/apache/flink/runtime/deployment/InputChannelDeploymentDescriptor.java

```diff
@@ -104,16 +107,17 @@ public String toString() {
 					(producerState == ExecutionState.RUNNING
 							|| producerState == ExecutionState.FINISHED)) {
 
-				final Instance partitionInstance = producerSlot.getInstance();
+				final TaskManagerLocation partitionTaskManagerLocation = producerSlot.getTaskManagerLocation();
+				final ResourceID partitionTaskManager = partitionTaskManagerLocation.getResourceID();
```

Comment:
Does it make sense to rename the variable to `producerTaskManager`? Then it is symmetric to `consumerTaskManager`.